libraries/chain/include/eosio/chain/name.hpp

```diff
@@ -53,7 +53,7 @@ namespace eosio::chain {
       return n;
    }
 
-   /// Immutable except for fc::from_variant.
+   /// Immutable except for fc::from_variant and istream.
```

Comment:
The original comment is incorrect since name has an implicit assignment operator. Probably should remove this comment.

Reply:
will do, thanks!